src/main/groovy/org/ysb33r/gradle/gradletest/TestGenerator.groovy

```diff
@@ -306,6 +321,90 @@ class TestGenerator extends DefaultTask {
      */
     @CompileDynamic
     private void copy(
+            final File targetDir,
+            final String testBase,
+            final String defaultTask,
+            final File manifestFile,
+            final File workDir,
+            final TestDefinition testDefinitions,
+            final List<String> arguments,
+            final boolean willFail,
+            final boolean deprecationMessageMode
+    ) {
+        copySub(testDefinitions.groovyBuildFiles,
+                GROOVY_TEST_POSTFIX,
+                targetDir,
+                testBase,
+                defaultTask,
+                manifestFile,
+                workDir,
+                testDefinitions,
+                arguments,
+                willFail,
+                deprecationMessageMode
+        )
+
+        copySub(testDefinitions.kotlinBuildFiles,
+                KOTLIN_TEST_POSTFIX,
+                targetDir,
+                testBase,
+                defaultTask,
+                manifestFile,
+                workDir,
+                testDefinitions,
+                arguments,
+                willFail,
+                deprecationMessageMode
+        )
+
+    }
+
+    private void copySub(
```

Comment:
this method takes the array of build objects and the file type "postfix" and reprocesses them into a format that the ORIGINAL copy method can use to build the spec files.